cluster/manifests/stackset-controller/01-stackset-crd.yaml

```diff
@@ -596,6 +596,25 @@ spec:
                         - maxReplicas
                         - metrics
                         type: object
+                      configurationResources:
```

Comment:
These CRD changes should also be guarded by the `stackset_configmap_support_enabled` config item.